Tests/Xpring/XpringClientTest.swift

```diff
@@ -154,4 +154,27 @@ final class XpringClientTest: XCTestCase {
 
     self.wait(for: [completionCalledExpectation], timeout: 10)
   }
+
+  func testMismatchedNetworks() throws {
+    // GIVEN a PayIDClient and an XRPClient on different networks.
+    let expectedTransactionHash = "deadbeefdeadbeefdeadbeef"
+    let xrpClient: XRPClientProtocol = FakeXRPClient(
+      network: .test,
+      getBalanceValue: .success(fakeBalanceValue),
+      paymentStatusValue: .success(fakeTransactionStatusValue),
+      sendValue: .success(expectedTransactionHash),
+      latestValidatedLedgerValue: .success(fakeLastLedgerSequenceValue),
+      rawTransactionStatusValue: .success(fakeRawTransactionStatusValue),
+      paymentHistoryValue: .success(fakePaymentHistoryValue),
+      accountExistsValue: .success(fakeAccountExistsValue)
+    )
+
+    let fakeResolvedPayID = "r123"
```

Comment:
fakeResolvedPayID makes me thing it'll be PayID format, my impression is that this is supposed to be an XRP classic address (starts with r, random stuff after). I'd find it helpful if this variable was renamed